en/setup/dev_env_linux_ubuntu.md

```diff
@@ -244,53 +244,10 @@ sudo apt-get remove gcc-arm-none-eabi gdb-arm-none-eabi binutils-arm-none-eabi g
 sudo add-apt-repository --remove ppa:team-gcc-arm-embedded/ppa
 ```
 
-Execute the script below to install 5.4:
-
-```sh
-pushd .
-cd ~
-wget https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2
-tar -jxf gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2
-exportline="export PATH=$HOME/gcc-arm-none-eabi-5_4-2016q2/bin:\$PATH"
-if grep -Fxq "$exportline" ~/.bash_profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
-popd
-```
-
-Run these commands to install the 32 bit support libraries (this might fail and can be skipped if running a 32 bit OS):
-
-```sh
-sudo dpkg --add-architecture i386
-sudo apt-get update
-
-sudo apt-get install libc6:i386 libgcc1:i386 libstdc++5:i386 libstdc++6:i386
-sudo apt-get install gcc-4.6-base:i386 
-```
-
-Now restart your machine.
+<!-- import GCC toolchain common documentation -->
+{% include "_gcc_toolchain_installation.txt" %}
```

Comment:
This imports same instructions as every other build. We can do this now that the 32 bit support libraries are no longer required.